packages/common/src/services/AccessService.ts

```diff
@@ -0,0 +1,51 @@
+import { inject, injectable } from 'inversify';
+
+import type { Passport } from '../../types/passport';
+
+import StorageService from './StorageService';
+
+const PASSPORT_KEY = 'passport';
+
+@injectable()
+export default class AccessService {
+  private readonly storageService: StorageService;
+
+  constructor(@inject(StorageService) storageService: StorageService) {
+    this.storageService = storageService;
+  }
+
+  generatePassport = async (host: string, siteId: string, jwt?: string): Promise<Passport | null> => {
```

Comment:
Can `host` be a private field in AccessService instead of being passed in? Seems like it can be encapsulated here and should be fairly much static.

Reply:
That would mean to use the config store in the service, as the host is available in the settings of the config. Are we okay with that?